languages/java/exercises/concept/arrays/.meta/src/reference/java/BirdCount.java

```diff
@@ -0,0 +1,57 @@
+
+class BirdCount {
+    private int[] birdsPerDay;
+
+    public BirdCount(int[] birdsPerDay) {
+        this.birdsPerDay = birdsPerDay;
+    }
+
+    public int[] getLastWeek() {
+        return birdsPerDay;
+    }
+
+    public int getToday() {
+        if (birdsPerDay.length == 0) {
+            return 0;
+        }
+        return birdsPerDay[birdsPerDay.length - 1];
+    }
+
+    public int incrementTodaysCount() {
+        birdsPerDay[birdsPerDay.length - 1] += 1;
+        return birdsPerDay[birdsPerDay.length - 1];
+    }
+
+    public boolean hasDayWithoutBirds() {
+        for (int day : birdsPerDay) {
+            if (day == 0) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public int getCountForFirstDays(int numberOfDays) {
+        int total = 0;
+
+        if (numberOfDays > birdsPerDay.length) {
+            numberOfDays = birdsPerDay.length;
+        }
+
+        for (int i = 0; i < numberOfDays; i++) {
+            total += birdsPerDay[i];
```

Comment:
The `+=` operator is also syntax that has not been introduced to the student. This issue also applies to the C# track, and I've [opened an issue for it](https://github.com/exercism/v3/issues/2112).